### DISCUSSION TITLE
Is it possible to "Configure with CMake Debugger" via Remote Development?

### DISCUSSION THREAD

**Discussion by Kentzo** [microsoft/vscode-cmake-tools#4133] [Q&A] [answered]
I'm trying to configure a project inside a Dev Container, but Configure with CMake Debugger" is not offered there (it's available locally). Is it a known limitation?

**Comment by gcampbell-msft** [comment, marked as answer]
@Kentzo To my knowledge, it shouldn't be a limitation. What version of cmake.exe are you using? We only show the command if that cmake.exe version supports debugging.

**Reply by Kentzo** [reply]
Indeed, the installed version was too old.
